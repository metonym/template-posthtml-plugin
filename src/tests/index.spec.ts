import posthtml from 'posthtml';
import { plugin } from '../';

describe('PostHTML plugin', () => {
  test('plugin matches snapshot', () => {
    posthtml()
      .use(plugin())
      .process(`<body></body>`)
      .then((result: { html: string }) => {
        expect(result.html).toMatchSnapshot();
      });
  });
});
