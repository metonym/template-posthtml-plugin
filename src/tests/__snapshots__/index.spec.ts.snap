// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostHTML plugin plugin matches snapshot 1`] = `"<body></body>"`;
